.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: node_js
+git:
+  depth: false
+
+dist: trusty
+sudo: required
+
+matrix:
+  include:
+    - node_js: 8
+      env: TEST_SUITE=unit
+      install: npm i
+      script: npm run test:unit-ci
+
+    - node_js: 8
+      env: TEST_SUITE=screenshots
+      install: npm i
+      script: npm run test:image-diff-ci
+      after_script:
+        - echo $SERVICE_ACCOUNT_KEY > ./key.json
```

Comment:
Can you not just reference SERVICE_ACCOUNT_KEY directly? Why do you have to write it to key.json?

Reply:
From digging into the docs, it looks like there's support for passing the key directly as a `credentials` attribute instead of a `keyFilename` attribute.

I'll update to use that.